backend/src/controllers/course.js

```diff
@@ -1,5 +1,5 @@
-const courseModel = require('../models/course')
-const questionModel = require('../models/question')
+const courseModel = require('../models/course')()
```

Comment:
So since we are not passing in a db, the test db is used, if I'm reading this correctly?

Reply:
when we are not passing in a db we use the singleton. Else we use the one we passed in.